code/modules/food_and_drinks/drinks/drinks/cans.dm

```diff
@@ -41,14 +41,14 @@
 	qdel(src)
 	return crushed_can
 
-/obj/item/reagent_containers/food/drinks/cans/AltClick(mob/user)
+/obj/item/reagent_containers/food/drinks/cans/CtrlClick(mob/user)
 	var/mob/living/carbon/human/H
 	if(!can_shake || !ishuman(user))
-		return ..()
+		return
```

Comment:
if you remove all calls to the parent function, nobody will be able to ctrl click pull this object

Reply:
True, forgot to do that after deciding halfway to use CtrlClick.